docs/api/en/core/BufferGeometry.html

```diff
@@ -40,6 +42,33 @@ <h2>Code Example</h2>
 		const mesh = new THREE.Mesh( geometry, material );
 		</code>
 
+		<p>Each three contiguous `vertices` positions represent a single triangle. Therefore, a same vertex can appears in multiple triangles, like: `1.0,  1.0,  1.0` (`v2`, `v3`) and `-1.0, -1.0,  1.0` (`v0`, `v5`).</p>
```

Comment:
>  Therefore, a same vertex can appears in multiple triangles, like

This sentence sounds wrong to me. In a triangle soup (a non-indexed buffer), a vertex is only part of a single triangle.

Reply:
you are right, I've changed it